tests/unit_tests/utils/test_names.py

```diff
@@ -75,3 +75,10 @@ def test_translate_simtools_to_corsika():
     simtools_pars = ["corsika_obs_level", "corsika_sphere_radius", "corsika_sphere_center"]
     for step, simtools_par in enumerate(simtools_pars):
         assert names.translate_simtools_to_corsika(simtools_par) == corsika_pars[step]
+
+
+def test_sanitize_name():
+    assert names.sanitize_name("y_edges unit") == "y_edges_unit"
+    assert names.sanitize_name("Y_EDGES UNIT") == "y_edges_unit"
+    assert names.sanitize_name("123name") == "_123name"
```

Comment:
Is this standard / usual behaviour to add a "_" for strings starting with a digit? Just asking, as I haven't seen it before.

Reply:
Good question. To use all the digits passed and in the same order, we have to add some character before the name. This functionality is not strictly necessary, because we don't expect names starting with numbers but it makes the function more complete (maybe more useful in the future).